legacy_promotions/lib/solidus_legacy_promotions/engine.rb

```diff
@@ -6,6 +6,10 @@ module SolidusLegacyPromotions
   class Engine < ::Rails::Engine
     include SolidusSupport::EngineExtensions
 
+    initializer "solidus_legacy_promotions.patch_state_machine", after: "spree.load_config_initializers" do
+      config.to_prepare { SolidusLegacyPromotions::SpreeOrderStateMachineDecorator }
```

Comment:
Would like to leave a comment here on why this is necessary

Reply:
I took this change out for now, as it only really relates to loading patches/decorators using Flickwerk - and we're not doing that yet.